plugins/scale/scale.cpp

```diff
@@ -1370,4 +1374,23 @@ class wayfire_scale : public wf::plugin_interface_t
     }
 };
 
-DECLARE_WAYFIRE_PLUGIN(wayfire_scale);
+class scale_combined : public wayfire_scale
```

Comment:
A cool trick ;)

I think it should be fine to just call the constructor and destructor directly from scale though. There shouldn't be any other direct calls ofc.